blockstore/settings/production.py

```diff
@@ -13,7 +13,7 @@
 LOGGING['handlers']['local']['level'] = 'INFO'
 
 CONFIG_FILE = get_env_setting('BLOCKSTORE_CFG')
-with open(CONFIG_FILE, encoding='utf-8') as f:
+with open(CONFIG_FILE) as f:
```

Comment:
What was fixed by removing `, encoding='utf-8'`?

Reply:
There was an exception raised while running migrations: 
`TypeError: 'encoding' is an invalid keyword argument for this function`
Removing the argument solved it. Also verified in other service repositories, like [this](https://github.com/open-craft/ecommerce/blob/master/ecommerce/settings/production.py#L47) for example, and saw that this wasn't used.